Cargo.toml

```diff
@@ -2,9 +2,18 @@
 
 members = [
     "vercel_runtime",
-    "examples/*",
+    "examples/simple",
+    "examples/cron",
+    "examples/nextjs",
 ]
 
 exclude = [
-    "test/*",
```

Comment:
This was buggy with rust-analyzer, so dropping the glob again.

Reply:
Should possibly have been `test/fixtures/*` ?

Reply:
Also tried that, kept complaining about it:
![CleanShot 2023-04-05 at 10 01 57](https://user-images.githubusercontent.com/8146736/230019113-fa6d211c-d7fb-4b59-9df7-c33accd64a24.png)